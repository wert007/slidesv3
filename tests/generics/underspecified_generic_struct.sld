func main() {
    let l = new List();
    l.add(5);
    l.add(9);
}