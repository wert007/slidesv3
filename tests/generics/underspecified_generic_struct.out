Error in ..\tests\generics\underspecified_generic_struct.sld:2:13: Struct List is a generic struct, but its generic type is undeclared.
